src/nunit.analyzers/Extensions/AttributeArgumentSyntaxExtensions.cs

```diff
@@ -69,35 +59,23 @@ internal static class AttributeArgumentSyntaxExtensions
                     {
                         canConvert = argumentType.SpecialType == SpecialType.System_String;
                     }
-
-                    if (canConvert)
+                    // Intrinsic type converters
+                    // https://github.com/dotnet/runtime/blob/master/src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/ReflectTypeDescriptionProvider.cs
+                    else if (targetType.ContainingNamespace?.Name == "System" && (
+                        targetType.Name == "CultureInfo" ||
+                        targetType.Name == "DateTimeOffset" ||
```

Comment:
Looking at the linked file for the Intrinsic type converters also lists
`DateTime` and `Version`. Should those also be added?

Reply:
`DateTime` was already covered by the previous `else if` case, but I think we could add `Version` (but I'll be doing it in a newly created PR based on the suggestions in https://github.com/nunit/nunit.analyzers/issues/196#issuecomment-611788092